CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 Mapbox welcomes participation and contributions from everyone.
 
 # main
+## Features ✨ and improvements 🏁
+* Add checks while applySettings to plugins. Only apply setting when setting a new value. ([#1189](https://github.com/mapbox/mapbox-maps-android/pull/1189))
```

Comment:
Maybe rephrase to highlight the main idea of the change, e.g.
```
* Optimize how plugins handle settings changes. Call `applySettings` only when settings value changes. 
```
?

Reply:
Sounds good.